internal/pkg/template/templates/environment/cf.yml

```
@@ -642,3 +642,6 @@ Outputs:
     Description: The ID of the Copilot-managed EFS filesystem.
     Export:
       Name: !Sub ${AWS::StackName}-FilesystemID
+  DeploymentController:
```

What do you think of naming this out like this?
```suggestion
  LastForceDeployID: 
```